<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValidationDock</class>
 <widget class="QDockWidget" name="ValidationDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Validointi</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="validate_button">
        <property name="text">
         <string>Validoi kaavasuunnitelma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="validate_plan_matter_button">
        <property name="text">
         <string>Validoi kaava-asia</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="validation_label">
      <property name="text">
       <string>Validointivirheet:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progress_bar">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ValidationTreeView" name="validation_result_tree_view"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValidationTreeView</class>
   <extends>QTreeView</extends>
   <header>arho_feature_template.gui.components.validation_tree_view</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
